src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -175,7 +175,9 @@ private IEnumerable<NuGet.Credentials.ICredentialProvider> GetCredentialProvider
         {
             var extensionLocator = new ExtensionLocator();
             var providers = new List<Credentials.ICredentialProvider>();
-            var pluginProviders = new PluginCredentialProviderBuilder(extensionLocator, Settings).BuildAll().ToList();
+            var pluginProviders = new PluginCredentialProviderBuilder(extensionLocator, Settings, Console)
+                .BuildAll(Verbosity.ToString())
```

Comment:
I discussed this offline with @pspill  NuGet.Common is shared by the credential provider, but LogLevel there is different from NuGet.exe, ex: it has Debug instead of Detailed, and 6 levels instead of 3.

I think for now keeping this as a string is fine. What would need to happen to improve this is to make a new assembly for nuget.exe extensions to build on that contains some common types.